source/administration/devops-command-center.rst

```diff
@@ -1,7 +1,177 @@
-==============================
 Mattermost Incident Management
 ==============================
 
-Better response to an incident is one of three ways that Mattermost Incident Management helps you provide more reliable services, in addition to gaining insights with incident reports and incorporating learnings with playbooks.
+*Available in Mattermost Enterprise Edition E20, Mattermost Cloud Professional, and Mattermost Cloud Enterprise.*
+
+Incidents are situations which require an immediate response and benefit from a clearly defined process guiding towards resolution. Mattermost Incident Management allows your team to coordinate, manage, and resolve incidents from within Mattermost. 
+
+Better response to an incident helps you provide more reliable services, in addition to gaining insights with incident reports and incorporating learnings with playbooks.
+
+- Automatically create a new channel that can be organized in the left-hand sidebar using custom categories.
+- Use playbooks to perform automated actions such as create a Jira ticket, start a Zoom call, or find out who is on-call in Opsgenie.
+- Iterate on and refine processes after each incident.
+
+When incidents are monitored, coordinated, and measured effectively, you can add transparency, maximize effectiveness, and save costs by cutting down time taken to respond to and resolve incidents.
+
+.. contents::
+  :depth: 2
+  :local:
+  :backlinks: entry
+  
+API Documentation
+-----------------
+
+The Mattermost Incident Management API specification is available `here <https://github.com/mattermost/mattermost-plugin-incident-management/blob/master/server/api/api.yaml>`_.
```

Comment:
@agarciamontoro, is there any more information we can add here? Maybe a link to any help wanted tickets?

Reply:
I think it's ok for now. Maybe we could say that the documentation is a work in progress? Up to you!